sdcm/tester.py

```diff
@@ -568,6 +568,7 @@ def get_cluster_gce(self, loader_info, db_info, monitor_info):
 
     def get_cluster_aws(self, loader_info, db_info, monitor_info):
         # pylint: disable=too-many-locals,too-many-statements,too-many-branches
+        ec2 = boto3.resource('ec2')
```

Comment:
no `region_name` parameter specified, this won't work for nondefault regions

Reply:
Fixed.
Testing the new solution.

Reply:
tests have passed, ready to merge